PoC-Projects.md

```diff
@@ -9,3 +9,4 @@ These are some ideas for proof of concept projects to explore schnorr and/or tap
  * Taproot/Schnorr support for python-bitcoinlib (etc?)
  * ...
 
+Note: Optech has developed a [Schnorr & Taproot library](https://github.com/bitcoinops/taproot-workshop), which can be used to prototype many of the proof-of-concept projects suggested above. The library features a Schnorr and MuSig implementation as well as methods to construct Taproot outputs from various types of Tapscripts. The documentation is written in the form of interactive Jupyter notebooks, which provide thorough examples on how to use the library.
```

Comment:
@jachiang - optech style guide is to lowercase schnorr and taproot (see https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#common-nouns)

Reply:
@jnewbery - Just for my understanding, this extends to all following expressions: musig, taproot, taptree, tapscript as well?